modules/zabbix/manifests/init.pp

```diff
@@ -25,7 +21,7 @@
     gpgkey      => 'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-ZABBIX',
```

Comment:
gpgcheck => 0 (false) but you are using gpgkey => 'file:///etc/pki/rpm-gpg/RPM-GPG-KEY-ZABBIX', how could it be?